.travis.yml

```
@@ -23,3 +23,4 @@ script:
   - multiqc -m star -o tests/multiqc_report_dev -t default_dev -k json --file-list data/special_cases/file_list.txt
   - multiqc -f empty_dir
   - multiqc -f data/modules/gatk/BaseRecalibrator/recal_data.table
+  - (cd ../test && multiqc ../test_data/data/ && python print_csp.py)
```

Can move the `multiqc --lint data/modules/` test command to be the last in the list before this one (add `-f` to that test command) and then you don't need to re-run MultiQC.

It's quite time-consuming to run on all of the test data, so better to avoid doing it twice if we can.
```suggestion
  - python ../test/print_csp.py
```

Would need to make the python script accept a parameter for the file path to use to make this work I guess. But not much code for a single argument like this.